<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PretWidget</class>
 <widget class="QWidget" name="PretWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="assuranceGroupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Assurance du prêt</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>sur capital emprunté</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>cpe rest du annuel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>sur capital restant du</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="emprunteur1">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Emprunteur #1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Taux</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="assTaux1">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Quotité</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="assQuotite1">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="emprunteur2">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Emprunteur #2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Taux</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="assTaux2">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Quotité</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="assQuotite2">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ou Coût total des assurances</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="montantAss"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>176</height>
    </rect>
   </property>
   <property name="title">
    <string>Caractéristiques du prêt</string>
   </property>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>145</y>
      <width>127</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de débloquage</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTaux">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>60</y>
      <width>135</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Taux d'intérêt annuel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDuree">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Durée du prêt (en mois)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMensualite">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>115</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ou Montant des mensualités</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mensualite">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>115</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duree">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tauxPret">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">0.045</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateDebloquage">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>145</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="capitalEmprunte">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>120000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCapitalEmprunte">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Capital emprunté</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupeEvenement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>701</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Evénements</string>
   </property>
   <widget class="QTableView" name="evenementsTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonAjouterEvenement">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resources/simulpret-resources.qrc">
      <normaloff>:/simulpret/icons/plus.png</normaloff>:/simulpret/icons/plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonSupprimerEvenement">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resources/simulpret-resources.qrc">
      <normaloff>:/simulpret/icons/delete.png</normaloff>:/simulpret/icons/delete.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView  {
     alternate-background-color: grey;
     background: white;
 }</string>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupeCoutCredit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Coût du crédit</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>143</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Coût total :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCoutTotal">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>200</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Coût de l'assurance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCoutAssurance">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>187</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Total des intérêts:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTotalInterets">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="frais">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>31</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frais bancaires</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>frais</zorder>
   <zorder>label_8</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QPushButton" name="calculerButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>170</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>calculer</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/simulpret-resources.qrc"/>
 </resources>
 <connections/>
</ui>
